apisix/plugins/consumer-restriction.lua

```diff
@@ -105,7 +106,11 @@ local function is_method_allowed(allowed_methods, method, user)
 end
 
 local function reject(conf)
-    return conf.rejected_code, { message = "The " .. conf.type .. " is forbidden." }
+    local reject_message = "The " .. conf.type .. " is forbidden." ;
```

Comment:
We can check `conf.reject_message` before creating new string?

Reply:
ok

Reply:
> 

ok